mne/viz/ica.py

```diff
@@ -143,7 +144,7 @@ def _plot_ica_properties(pick, ica, inst, psds_mean, freqs, n_trials,
 
     # image and erp
     plot_epochs_image(epochs_src, picks=pick, axes=[image_ax, erp_ax],
-                      combine=None, colorbar=False, show=False, **image_args)
+                      combine=None, colorbar=False, show=False, **image_args, dropped_indices=dropped_indices)
```

Comment:
I don't know if adding new args to `plot_epochs_image` is a good idea - the use case here seems to be very specific.

Reply:
Yep, I will try to directly edit the ```epochs_src``` and change how NaNs are handled in the function :)